vault/src/cancellation.rs

```diff
@@ -391,10 +389,11 @@ mod tests {
         }
     }
 
+    use subxt::utils::Static;
     fn default_issue_request() -> InterBtcIssueRequest {
         InterBtcIssueRequest {
             amount: Default::default(),
-            btc_address: Default::default(),
+            btc_address: Static(Default::default()),
```

Comment:
Why is this required?

Reply:
`Address` enum in node primitives use  `sp_core::H160` & `sp_core::H256`, which don't implement `scale_encode` & `scale_decode` traits. Therefore static is needed another option discussed was to use `subxt::sp_core` which is feature gated by `substrate-compact` feature.